core/src/main/java/org/apache/carbondata/core/datastore/impl/btree/BTreeNonLeafNode.java

```diff
@@ -24,8 +24,8 @@
 import org.apache.carbondata.core.datastore.DataRefNode;
 import org.apache.carbondata.core.datastore.FileHolder;
 import org.apache.carbondata.core.datastore.IndexKey;
-import org.apache.carbondata.core.datastore.chunk.DimensionColumnDataChunk;
-import org.apache.carbondata.core.datastore.chunk.MeasureColumnDataChunk;
+import org.apache.carbondata.core.datastore.chunk.impl.DimensionRawColumnChunk;
```

Comment:
Why putting `DimensionRawColumnChunk` under `impl` package? It is exposed like interface right?